docs/samples/Microsoft.ML.Samples/Microsoft.ML.Samples.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <OutputType>Exe</OutputType>
-    <SignAssembly>false</SignAssembly>
-    <PublicSign>false</PublicSign>
+    <AssemblyName>Samples</AssemblyName> 
```

Comment:
Is there any reason to make the assembly name distinct from the project name? I got the point with the console MAML since that was intended to be used, but I don't think it's a good idea here.

Reply:
If you want to rename the AssemblyName, just rename the `.csproj` file instead.